docs/src/what-is-a-validator.md

```diff
@@ -2,7 +2,7 @@
 title: What is a Validator?
 ---
 
-A validator is a computer that helps to run the Solana network. Each validator executes a program that keeps track of all accounts on the Solana cluster and validates transactions being added to the network. Without validators, Solana would not be able to function.
+A validator is a computer that helps to run the Solana network. Each validator executes a program that keeps track of all accounts on the Solana cluster and validates transactions being added to the network. Without validators, Solana would not be able to function. Agave is one of several possible validator clients developers can use to help run the Solana network.
```

Comment:
"developers" might be misleading here. Maybe "operators", or switch to passive voice: "that can be used to help..."